buildinfo_test.go

```diff
@@ -576,8 +587,12 @@ func TestReadGitConfig(t *testing.T) {
 	if !strings.HasSuffix(url, ".git") {
 		url += ".git"
 	}
-
 	assert.Equal(t, url, gitManager.GetUrl(), "Wrong url")
+
+	branch, _, err := gitExecutor.GetBranch()
+	require.NoError(t, err)
```

Comment:
```suggestion
	assert.NoError(t, err)
```
Please consider using `assert` over `require` because when `require` fails, it stops the execution of the tests. This will skip the test's cleanup.
Could you please change also other `require` to `assert` in `TestReadGitConfig`?